CONTRIBUTING.md

```diff
@@ -23,40 +23,100 @@ Anyone interested in contributing should join the mailing list and other
 
 ### Reviewing Pull Requests
 
-If you are new to the group, reviewing pull requests and commenting on issues
-is a great way to get involved!
+If you are new to the group, reviewing pull requests and commenting on issues is
+a great way to get involved!  We strongly encourage and support all our members
+to participate in anyway they can.
 
-Except for urgent or very small grammar or spelling fixes, we leave pull 
-requests open for at least 24 hours, so that others have the chance to 
+Except for urgent or very small grammar or spelling fixes, we leave pull
+requests open for at least 24 hours, so that others have the chance to
 review/comment.   
 
-Anyone who reviews a pull request should leave a note to let others know that 
-someone has looked at it.  Any change should have a +1 from someone else.
-Larger changes should have multiple reviewers.
+#### Favorable review
```

Comment:
i'm not quite sure i understand the motivation of this section. are we adding a mechanism of favorable reviews for non write access members that acts as a non-official github approval?

Reply:
So we've always had this in some fashion.  Anyone can review a PR and +1 it to say it looks good.  It lets a person with merging power know someone besides them has at least looked it over.  
The intent of this is to provide context around what is considered a 'good' review of the PR.

Reply:
i think there may be a mechanisms to allow non-write users to do things like this with github bots, could be something to look at in the future.